Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM python:3
```

Comment:
You mean the `python:3` image in prow CI, or building an image from `Dockerfile` (https://github.com/openshift/release/pull/5123)?

Reply:
@estroz I'm talking about the fact that there is an edit to the dockerfile located here https://github.com/operator-framework/operator-courier/blob/master/Dockerfile in this pr. This dockerfile's purpose is as a way for local users to build and run courier locally without having python3 installed on their environments. It is not currently being used to build or publish any container image in any fashion, CI or otherwise.

Since this pr is editing the file, I want to understand if this dockerfile is being used for a different purpose now and what that purpose is.

Reply:
It is, although the change has nothing to do with its use as a [base image](https://github.com/openshift/release/pull/5123/files#diff-8a89a5bc72b095f512ec62ee2e24427dR13) in CI, just [`COPY` vs `ADD`](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) best practices.

Reply:
> It is, although the change has nothing to do with its use as a [base image](https://github.com/openshift/release/pull/5123/files#diff-8a89a5bc72b095f512ec62ee2e24427dR13) in CI, just [`COPY` vs `ADD`](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) best practices.

Okay, so if updating this dockerfile is unrelated to this change can you please remove it from this pr since this change (while valid) is noise? I would prefer that all the changes in a given PR are related to the purpose described so that we have a clear history and can revert if needed without losing unrelated changes.